website/static/js/anonymousLogActionsList.json

```diff
@@ -17,9 +17,9 @@
     "edit_description" : "A user edited the description of a project",
     "updated_fields" : "A user changed the ${updated_fields} for a project",
     "external_ids_added" : "A user created external identifiers on a project",
-    "citation_custom_added" : "A user created a custom citation for a project",
-    "citation_custom_edited" : "A user updated a custom citation in a project",
-    "citation_custom_removed" : "A user removed a custom citation in a project",
+    "custom_citation_added" : "A user created a custom citation for a project",
+    "custom_citation_edited" : "A user updated a custom citation for a project",
+    "custom_citation_removed" : "A user removed a custom citation from a project",
```

Comment:
Didn't add tests for these - we're not generally adding tests that all these js pieces format correctly, and I think it's overkill to test these individually.  If it breaks, you see a "Unable to retrieve log details message," we don't expose data we shouldn't, etc. etc.  A future improvement could be something that requires all logs in `logActionsList.json` to `anonymousLogActionsList.json`.